src/js/components/DataTable/DataTable.js

```diff
@@ -219,62 +223,35 @@ const DataTable = ({
     console.warn('DataTable cannot combine "size" and "resizeble".');
   }
 
+  const [page, showItem] = normalizeShow(showProp, 'dataTable');
+  const [setPage, currentItems, currentPage] = usePagination({
```

Comment:
Something feels a bit confusing to have both `page` and `currentPage`.

Reply:
I think we can clean up a bit here by adjusting variable naming. I'll do an iteration and see if it's more clear.

Reply:
Cleaned this up.